packages/neuron-ui/src/components/TransactionFeePanel/transactionFeePanel.module.scss

```diff
@@ -21,20 +21,20 @@
 .dropdown {
   margin-top: 8px;
   position: relative;
-  --tag-red-bg-color: #FFE8E8;
-  --tag-green-bg-color: #EFFAF7;
-  --tag-blue-bg-color: #E8FAFF;
+  --tag-red-bg-color: #ffe8e8;
```

Comment:
@yanguoyu How about we use a `vscode setting file` to constrain the formatting of the entire project, in the future? Such as CSS color formatting, single quotes and double quotes, and so on. Because when I format it locally, I get this problem as well. The color value will be lowercase, but not necessarily for other colleagues maybe.

Reply:
Can we set these rules by prettier? I think `vscode` setting is more about the local setting.

Reply:
> Can we set these rules by prettier? I think `vscode` setting is more about the local setting.

 Sounds good.